test/integration/test-with-k3s-setup-ee.sh

```diff
@@ -38,6 +38,11 @@ export INFERENCE_FLAVOR="CPU"
 export LIVE_TEST_ENDPOINT="http://localhost:$EDGE_ENDPOINT_PORT"
 export REFRESH_RATE=60 # not actually different than the default, but we may want to tweak this
 
+# Compute the image tag name before we muck with the config file so we get
+# the tag that will correspond to the current commit so it can match the image
+# that was built and pushed to ECR
+export IMAGE_TAG=$(./deploy/bin/git-tag-name.sh)
```

Comment:
Does this mean that if we run this test twice, the second time it wont grab the same image? I suppose this doesnt matter if we're using this only in CICD.

Reply:
It should grab the same image both times. The issue is that the way Roxanne set up the test, it modifies the edge-config file in place for the duration of the test. If we run the git-tag-name.sh after that we get a tag name that has never existed before and, therefore, there can't really be an image for this. So I want to be sure that I grab the tag first.

She restores the original file after the test, so a second run should get the same tag.

This isn't the most robust thing. But we're going to deprecate this whole `setup-ee.sh` mechanism RSN, so I'm not too worried about it. For the helm version of the test, I use a file in `/tmp`  to avoid this problem and just because it seems better.